### ISSUE TITLE
SingleBoostr not detecting all games

### ISSUE THREAD

**Issue by Zwero1** [Ni1kko/HourBoostr#25]
### When using the GUI not all of my games show up on the list. I don't know if it's specific to the games or me but the games are:
Spore
Dyson Sphere Program
OMORI
Black Mesa
Terraria
Poly Bridge

**Comment by alucard12345678** [comment]
Zup! F
Zup! S

**Comment by Zwero1** [comment]
thanks

**Comment by alucard12345678** [comment]
any fix?

**Comment by Zwero1** [comment]
No, also both Zup! F and S work fine for me, so I guess for whatever reason it's happening it's different for every person
for now i just put my games in manually in the terminal version of the app.
